source/connection_manager.c

```diff
@@ -558,21 +610,19 @@ static void s_aws_http_connection_manager_build_transaction(struct aws_connectio
 
 static void s_aws_http_connection_manager_execute_transaction(struct aws_connection_management_transaction *work);
 
-static void s_aws_http_connection_manager_destroy(struct aws_http_connection_manager *manager) {
+static void s_aws_http_connection_manager_destroy_final(struct aws_http_connection_manager *manager) {
```

Comment:
Should this be like ```destroy_impl``` or ```destroy_common``` or something? This feels weird, especially since its the first thing you see with regards to destruction